redpanda/values.yaml

```diff
@@ -316,94 +316,104 @@ serviceAccount:
 # These values do not pertain to the kubernetes objects created with helm.
 # Instead these parameters get passed directly to the Redpanda binary at startup.
 # See https://docs.redpanda.com/docs/cluster-administration/configuration/
+# There are three subsections, cluster, tunable and node. The cluster and tunable sections
+# are written to the cluster .bootstrap.yaml and read into central config upon startup.
```

Comment:
Maybe this text should be taken out since it doesn't make sense for all Redpanda versions (older version wouldn't have `bootstrap.yaml` and would expect config to be written to `redpanda.yaml`).

Reply:
@vuldin - the additional and potentially confusing lines are now removed in [24e9ee1](https://github.com/redpanda-data/helm-charts/pull/117/commits/24e9ee1aa87bcf44d9c39e677403c3e3c626ab21)